.circleci/base_config.yml

```diff
@@ -261,6 +261,40 @@ jobs:
                   - 3rdParty/iresearch/tests/resources
                   - 3rdParty/rta-makedata
 
+  compile-mac:
+    parameters:
+      preset:
+        type: string
+      build-tests:
+        type: boolean
+      publish-artifacts:
+        type: boolean
+        default: true
+    macos:
+      xcode: 14.2.0
+    resource_class: macos.m1.medium.gen1
+    steps:
+      - checkout-arangodb:
+          with-submodules: true
+          destination: "~/project"
+      - run:
+          name: "Install"
+          command: |
+            brew install cmake openssl@3 llvm@14
+      - run:
+          name: "Configure"
+          command: |
+            export PATH="/opt/homebrew/opt/llvm@14/bin:$PATH"
+            export MACOSX_DEPLOYMENT_TARGET=13.2
```

Comment:
Too new deployment target. Should be adjusted with PM.

Reply:
x86-64 mac on CircleCI is already deprecated and being removed. So that would need to be handles in a different way.